src/resource/utils/fetch-resource.js

```diff
@@ -1,6 +1,5 @@
 import URL from 'url';
 import request from 'postman-request';
-import { Errors } from 'utils';
```

Comment:
If we're no longer uses `Errors` here, I think we should be able to get rid of that whole `utils/errors.js` file.